bloom-housing/bloom
# refactor: max width layout

## Description

A refactor based on Detroit work. We don't currently have a layout component unrelated to markdown content that sets up default page margin and padding. This PR adds that generic layout using Seeds tokens.

No related issue.

## How Can This Be Tested/Reviewed?

In the places where we are using MarkdownSection where we aren't using markdown in the content, I transition to the MaxWidthLayout component.

You can compare an invalid link like https://bloom.exygy.dev/bonk to this PR's preview to see a place where the new layout is being used.

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
